src/main/java/TaskList.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private final ArrayList<Task> myTaskList;
```

Comment:
Hi, I noticed a contrast where myTaskList has a final modifier, but myTaskList is directly modified in the later code by adding to the list. May I ask what the reason for adding the final modifier is?